pynetbox/core/query.py

```diff
@@ -268,7 +269,7 @@ def _make_call(self, verb="get", url_override=None, add_params=None, data=None):
             url_override or self.url, headers=headers, params=params, json=data
         )
 
-        if req.status_code == 204 and verb == "post":
+        if req.status_code in [204, 409] and verb == "post":
```

Comment:
This'll work, but I think I want to refactor this a bit at some point to not handle errors here.